packages/notifier/README.md

```diff
@@ -166,74 +173,70 @@ const {
 observeIteration(subscription, adapterPublication);
 consume(adapterSubscription);
 ```
-This will work for `subscription` being a reference to any AsyncIterable. If
-Alice only needs to consume in a lossy manner, she can use `makeNotifierKit()`
-instead, which will still work independent of what kind of AsyncIteratable
-`subscription` is a reference to.
+This works when `subscription` is a reference to an AsyncIterable. If Alice only needs to 
```

Comment:
```suggestion
This works when `subscription` is a reference to any AsyncIterable. If Alice only needs to 
```

"any" was correct. The point is that it works independent of what kind of AsyncIterable it is.